.github/workflows/node.yml

```diff
@@ -0,0 +1,36 @@
+name: Code Checker
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    # Install dependencies and lint the typescript using prettier
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
```

Comment:
Suggestion: Use Node 12.x as this is the LTS version.

We should also probably specify this in the `engines` section in `package.json`.

Reply:
Done!